definitions/tools/cellmatch_lineage.cwl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Running a script to identify lineage of cells"
+
+baseCommand: ["/usr/local/bin/Rscript", "/opt/CellMatch_Haemopedia.r"]
+
+requirements:
+    - class: DockerRequirement
+      dockerPull: "saimukund/seurat_docker"
```

Comment:
It'd be good to have an explicit version for docker images used in our CWL.  (As a side note, we usually avoid including the word `docker` in the name of docker images.)

Reply:
Did the changes